File: src/main/java/com/koteldlc/client/module/modules/visual/HurtTimeESP.java
package com.koteldlc.client.module.modules.visual;

import com.koteldlc.client.gui.settings.SliderSetting;

public class HurtTimeESP extends BaseEspModule {
    private final SliderSetting durationSec = register(new SliderSetting("duration_sec", 1.0, 0.5, 3.0));

    public HurtTimeESP() {
        super("HurtTimeESP", "Подсветка недавно получивших урон игроков", 0xFFFF3030);
        blink.setValue(true);
    }

    public double getDurationSec() {
        return durationSec.getValue();
    }
}

File: kotlin-mod/src/main/kotlin/com/koteldlc/client/modules/visual/HurtTimeESP.kt
package com.koteldlc.client.modules.visual

class HurtTimeESP : BaseEspModule("HurtTimeESP") {
    init {
        settings["color"] = 0xFFFF2D2D.toInt()
        settings["durationSec"] = 1.2
        settings["blink"] = true
    }

    override fun onRenderEsp() {
        // Подсветка игроков, получивших урон в последние durationSec секунд.
    }
}
